src/components/LoadingIndicator/LoadingIndicator.jsx

```diff
@@ -42,4 +42,5 @@ LoadingIndicator.propTypes = {
 
 LoadingIndicator.defaultProps = {
   setFocus: false,
+  label: "loading-indicator",
```

Comment:
Let's change this to just read "Loading". I've been chatting with @jbalboni and @noahgelman about possible fast-follow improvements for this component using a role="status" and requiring a `{message}` prop. That's out of the scope for this ticket, and they're still reviewing it in VAOS. Having this label gets this PR to where we need it, with axe-core errors cleaned up and plain language for the spinner.